Update view_participant_data.md (#2738)
DFE-Digital/register-early-career-teachers-public: app/views/api/guidance/guidance_for_lead_providers/view_participant_data.md

 
Providers can use the `GET /participants/{id}` endpoint to view individual records. 

### Participant induction 

> The API uses the following fields to describe a participant’s induction in the participant endpoints: 

Reviewer: So I think we should be careful here, as induction only applies to ECTs, not mentors. Mentors just do training, and still have the TRN and eligible for funding fields. They're not really related to induction at all for mentors.

Author: The previous design highlighted these fields as 'key', and they are important for ECT and mentor funding, and ECT induction start date.

I'll propose a slightly different heading that includes these fields, but will add text to clarify context and use.

* `eligible_for_funding`, shows if a participant is eligible for funding. Can be `true`, `false`, or `null`. 

> * `overal_induction_start_date`, the date an ECT officially began statutory induction as submitted by their appropriate body.

Reviewer 2: ```suggestion
* `overall_induction_start_date`, the date an ECT officially began statutory induction as submitted by their appropriate body.
```

* `induction_end_date`, the date an ECT completed (passed or failed) their induction.
* `teacher_reference_number`, the participant’s teacher reference number. If a TRN is missing, it means the participant’s registration predates validation, and we have no matching TRN in our records.

The following table explains what the `eligible_for_funding` values mean. 

| `eligible_for_funding` | Definition | 
| ------------ | ------------- | 

> | `true` | Set for ECTs and mentors who become eligible at any point. For example, when an ECT’s induction is recorded or a mentor is eligible to start training. A `true` status will remain once set. Other fields are used to indicate that training has stopped or to block participants from being registered including `induction_completion_date`, `mentor_funding_end_date`, or `reason`. | 

Reviewer: Other fields are used ot indicate training has stopped but also induction. I don't know if it should be spelt out separately these values for ECTs and mentors, in separate tables?

Author: Seems reasonable to seek a different approach if this isn't clear.  Will check in with @claire-hughez.

Author: Corrected 'reason' to provide full field name.

Split out the 'other fields' content from the table and added underneath while splitting ECT and mentor field descriptions.

> | `true` | Set for when ECTs and mentors who become eligible at any point. For example, when an ECT’s induction is recorded or a mentor is eligible to start training. A `true` status will remain once set. Other fields are used to indicate that training has stopped or to block participants from being registered including `induction_completion_date`, `mentor_funding_end_date`, or `reason`. | 

Reviewer 2: Is the "when" supposed to be there? 💭

```suggestion
| `true` | Set for ECTs and mentors who become eligible at any point. For example, when an ECT’s induction is recorded or a mentor is eligible to start training. A `true` status will remain once set. Other fields are used to indicate that training has stopped or to block participants from being registered including `induction_completion_date`, `mentor_funding_end_date`, or `reason`. | 
```

> | `false` | Set for new ECTs who are exempt from induction. `false` will never be set for mentor records. |  

Reviewer: new mentor records - I think we have some old ones potensh that failed validation in earlier cohorts

Author: Added 'new'

> | `null` | Set when ECT eligibility is not known. For example, before an induction is recorded by an appropriate body. `null` will never be set for mentor records. |  

Reviewer: new mentor records (I've have to double check if there are old ones - I'm not sure and not sure how long it will take to find out!)

Author: Added 'new'

 
Providers can use the `GET /participants/{id}` endpoint to view individual records. 

### Participant funding and induction 

> The API uses fields to describe funding eligibility for ECTS and mentors, and induction start and end dates for ECTs. 

Reviewer: should mentor funding end date and reasons be touched on, given the header?

Reviewer: lol, just seen it later

* `eligible_for_funding`, shows if an ECT or mentor is eligible for funding. Can be `true`, `false`, or `null`. 
* `overall_induction_start_date`, the date an ECT officially began statutory induction as submitted by their appropriate body.
* `induction_end_date`, the date an ECT completed (passed or failed) their induction.
* `teacher_reference_number`, the ECT or mentor's teacher reference number. If a TRN is missing, it means the participant’s registration predates validation, and we have no matching TRN in our records.

The following table explains what the `eligible_for_funding` values mean. 

| `eligible_for_funding` | Definition | 
| ------------ | ------------- | 
| `true` | Set for ECTs and mentors who become eligible at any point. For example, when an ECT’s induction is recorded or a mentor is eligible to start training. A `true` status will remain once set. | 
| `false` | Set for new ECTs who are exempt from induction. `false` will never be set for new mentor records. |  
| `null` | Set when ECT eligibility is not known. For example, before an induction is recorded by an appropriate body. `null` will never be set for new mentor records. |  

> Other fields are used to indicate that ECTs and mentors have stopped training or are not eligible for further funding. 

Reviewer: ```suggestion
Other fields are used to indicate that ECTs and mentors have completed induction or are not eligible for further funding. 
```

For mentors:

* `mentor_funding_end_date`
* `mentor_ineligible_for_funding_reason`

### Participant statuses 

The API uses two different status fields to describe a participant’s journey in the participant endpoints: 

* `training_status`, set by providers through the API. It determines what actions providers can take, such as updating data or submitting declarations.
* `participant_status`, shows if a participant is still engaging with training at their school. It moves to `left` whenever the school or lead provider indicates the participant is no longer training with the current lead provider, even if they remain at the school.

Together, these statuses give both providers and schools a shared view of a participant’s training progress. 

The following table explains what the training status fields mean. 

| `training_status` | Definition | Notes | 
| ------------ | ------------- | ------------- | 
| `active` | Participant is currently in training. | Update participant data and submit declarations. | 
| `deferred` | Participant has paused training. | Cannot update participant data. Can submit declarations. Providers must notify us when the participant resumes. | 

> | `withdrawn` | Participant has left training. | Cannot update participant data. Can only submit backdated declarations if `declaration_date` is before `withdrawal_date`. | 

Reviewer: @ethax-ross  is this true on declarations? I'm not sure

Reviewer 3: Yeah this is accurate; if you're deferred you can still receive a declaration, if you're withdrawn you can receive a declaration as long as the declaration date is earlier/on the withdrawal date (not sure if we want to call out the **on** or before here though?
